oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/AlarmCore.java

```diff
@@ -47,6 +47,14 @@
         return alarmRulesWatcher.getRunningContext().get(metricsName);
     }
 
+    /**
+     * Perform a series of tasks every 10 seconds, including update alarm settings dynamically,
+     * check alarm rules by check function, and calling doAlarm for alarm if trigger the
+     * rules (Send alarm message to webhook(s) through HTTP post).
+     * the operation of time windows(The time window is an important concept in stream computing,
+     * this is similar to the window in spark streaming, but only specially matches the own alarm
```

Comment:
Using another open source project to explain ours is very tricky and confusing. The better way is an ACIIS graph